app/lib/actions/katello/repository/destroy.rb

```diff
@@ -14,7 +14,7 @@ def plan(repository, options = {})
           action_subject(repository)
           plan_action(ContentViewPuppetModule::Destroy, repository) if repository.puppet?
           plan_action(Pulp::Repository::Destroy, pulp_id: repository.pulp_id)
-          plan_action(Product::ContentDestroy, repository) unless skip_environment_update
+          plan_action(Product::ContentDestroy, repository)
```

Comment:
Was this intended ?  Was this issue because we mistakenly assumed something along the lines of content getting automatically destroyed in CP ? Was some other action supposed to handle this?